message/validation/consensus_validation.go

```diff
@@ -80,13 +80,17 @@ func (mv *messageValidator) validateConsensusMessageSemantics(
 ) error {
 	signers := signedSSVMessage.GetOperatorIDs()
 	quorumSize, _ := ssvtypes.ComputeQuorumAndPartialQuorum(len(committee))
+	msgType := consensusMessage.MsgType
+
 	if len(signers) > 1 {
-		if consensusMessage.MsgType != specqbft.CommitMsgType {
+		// Rule: Decided msg with different type than Commit
+		if msgType != specqbft.CommitMsgType {
 			e := ErrNonDecidedWithMultipleSigners
 			e.got = len(signers)
 			return e
 		}
 
+		// Rule: Number of signers must be more than quorum size
 		if uint64(len(signers)) < quorumSize {
```

Comment:
@MatheusFranco99 Yes, I left a comment in the KB about this: https://github.com/ssvlabs/knowledge-base-internal/pull/20/files#r1608582666

It used to be here, but I wrote tests for all rules and I couldn't trigger this rule with a test because those two checks already cover it, so I had to remove it. Can we remove it from the KB?

Reply:
Yes